README.md

```diff
@@ -134,7 +134,11 @@ The full API Reference is available here.
 
 #### Full Examples: Object Operations
 * [PutObject.java](https://github.com/minio/minio-java/tree/master/examples/PutObject.java)
+* [PutObjectEncryptedAes.java](https://github.com/minio/minio-java/tree/master/examples/PutObjectEncryptedAes.java)
```

Comment:
The style is copied from google.  Not sure whether we need to touch these settings.